src/helper/service.cpp

```diff
@@ -72,6 +77,36 @@ service::ptr service::from_settings(const service_identifier &id,
         std::chrono::milliseconds{rc_settings.poll_interval});
 }
 
+void service::handle_error()
+{
+    rc_action_ = [this] { discover(); };
+    errors_++;
+    interval_ = std::chrono::duration_cast<std::chrono::milliseconds>(
+        poll_interval_ * pow(2, std::min((errors_ - 1), max_increment)));
```

Comment:
Our default poll interval is 1000ms so in theory this could result in a 17 minute interval, this seems too high, I would limit it to something along the lines of 1 to 5 minutes.

Reply:
If I've done the maths correctly, currently it stops on `8 minutes and a half`. I will decrease the max errors to 9 so it will stop at 4,2 minutes

Reply:
Actually I changed to 8 as I am doing now `errors_+` after calculating the interval to avoid the `errors_ -1`